### PR TITLE
Add webfont spoqa han sans

### PR DESCRIPTION
**Pull request by the1900**
This font is better than NotoSans and I think people will like it.

### CODE REVIEW

**Comment by the1900** [comment]
Why jsdelivr hasn't copied my spoqa-han-sans [project](https://github.com/the1900/spoqa-han-sans) to CDN?? wasn't it finish what I need to do ?

File: files/font-spoqa-han-sans/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "Spoqa (spoqa.com)"
+github = "https://github.com/the1900/spoqa-han-sans"
+homepage = "http://spoqa.github.io/spoqa-han-sans/"
```
Comment (megawac) [inline files/font-spoqa-han-sans/info.ini]: wait, why are you referencing a different website?

**Comment by megawac** [comment]
@the1900 why don't you submit these changes to the original spoqa repo instead of forking and taking spoqa's assets?

File: files/font-spoqa-han-sans/info.ini (hunk 1 above)
Comment (the1900) [inline files/font-spoqa-han-sans/info.ini]: Well I haven't thought about it.

Anyway, As you commented. I requested spoqa to add it in JSDelivr

**Comment by the1900** [comment]
@megawac  If you don't mind can you answer me?
I added info.ini and update.json. After this I've requested merge so are these all? even I don't use bower. I forked jsdelivr and add files on patch-1 branch so I think JSdelivr should copy my spoqa-han-sans repo. 

If you canceled merge becase it is not the original repo then please notify me. I want to know am I right to add files to JSDelivr CDN. I thought it is all automatic when using github wich contribute.md recommended Now I am confused.

**Comment by megawac** [comment]
@the1900 you can submit a project on behalf of another repo, its just odd to submit a fork of another project under the same name. I don't think we have precedent here

**Comment by the1900** [comment]
@megawac Oh this is what you said, right??

yeah I misunderstood the process. I thought only my repo could be submitted to JSDelivr.

**Comment by megawac** [comment]
Thanks looks good

**Pull request merged**